config/default/user.role.stanford_student.yml

```diff
@@ -1,11 +1,12 @@
 uuid: d68986fc-b911-4619-bffb-06937e66dd47
 langcode: en
 status: true
-dependencies: {  }
+dependencies: null
 _core:
   default_config_hash: utSJJ9cSylZEVzqaBr7cYV5wyXdC4xubL0L3QevhYRg
 id: stanford_student
 label: 'Stanford Student'
 weight: -3
 is_admin: null
-permissions: {  }
+permissions:
```

Comment:
No permissions should be for this role. it is meant to be just an identifier.

Reply:
Removed the permission, it was not being used because that functionality was reimplemented in another way.